src/mame/atari/atarist.cpp

```diff
@@ -1740,7 +1740,7 @@ static INPUT_PORTS_START( ikbd )
 	PORT_BIT( 0x08, IP_ACTIVE_LOW, IPT_KEYBOARD ) PORT_NAME("Tab") PORT_CODE(KEYCODE_TAB) PORT_CHAR('\t')
 	PORT_BIT( 0x10, IP_ACTIVE_LOW, IPT_KEYBOARD ) PORT_CODE(KEYCODE_Q) PORT_CHAR('Q')
 	PORT_BIT( 0x20, IP_ACTIVE_LOW, IPT_KEYBOARD ) PORT_CODE(KEYCODE_A) PORT_CHAR('A')
-	PORT_BIT( 0x40, IP_ACTIVE_LOW, IPT_KEYBOARD )
+	PORT_BIT( 0x40, IP_ACTIVE_LOW, IPT_KEYBOARD ) PORT_CODE(KEYCODE_PGDN) // ISO KEY
```

Comment:
If by “ISO KEY” you mean the key to the right of left shift, you’re supposed to use `KEYCODE_BACKSLASH2` for that.

Reply:
`IPT_KEYBOARD` inputs need names or characters to represent the keycap markings.

Reply:
> @galibert is refactoring this driver at the moment. It’s going to be changed completely fairly soon. It’s not worth doing conflicting minor changes like this when the driver is in its current sorry state.

Ok, I understand.  Would you like me to close the PR then?

Reply:
You have to target his branch if really ...